packages/verification-pool-api/src/verification.ts

```diff
@@ -70,18 +71,31 @@ function getFormattedMessage(message: string) {
   return message
 }
 
-async function sendViaTwilio(phoneNumber: string, messageText: string) {
+const NEXMO_COUNTRY_CODES = ['MX', 'US']
+
+async function sendViaTextProvider(phoneNumber: string, messageText: string) {
   try {
-    console.info('Sending message via Twilio')
-    await getTwilioClient().messages.create({
-      body: messageText,
-      from: twilioPhoneNum,
-      to: phoneNumber,
-    })
-    console.info('Message sent via Twilio')
+    console.info('Sending message via text provider')
+
+    const countryCode = phoneUtil.getRegionCodeForNumber(phoneUtil.parse(phoneNumber))
+    if (countryCode === undefined) {
+      throw new Error('Could not detect country code of ' + phoneNumber)
+    }
+
+    if (NEXMO_COUNTRY_CODES.indexOf(countryCode) === -1) {
+      await getTwilioClient().messages.create({
+        body: messageText,
+        from: twilioPhoneNum,
+        to: phoneNumber,
+      })
+      console.info('Message sent via Twilio')
+    } else {
+      await sendSmsWithNexmo(countryCode, phoneNumber, messageText)
+      console.info('Message sent via Nexmo')
+    }
   } catch (e) {
-    console.error('Failed to send twilio message', e)
-    throw new Error('Failed to send twilio message' + e)
+    console.error('Failed to send text message via txt provider', e)
+    throw new Error('Failed to send text message via txt provider' + e)
```

Comment:
Nit: this is probably my mistake but we shouldn't include `e` in the new error here

Reply:
I think including more error information than not is usually more helpful, so I personally actually like it

Reply:
I feel like it just adds noise given that we properly log the error in the line above but it's really nbd